virt/virtual_machines/virt-managing-vms-openshift-pipelines.adoc

```diff
@@ -8,7 +8,7 @@ toc::[]
 
 xref:../../cicd/pipelines/understanding-openshift-pipelines.adoc#understanding-openshift-pipelines[{pipelines-title}] is a Kubernetes-native CI/CD framework that allows developers to design and run each step of the CI/CD pipeline in its own container.
 
-The Tekton Tasks Operator (TTO) integrates {VirtProductName} with {pipelines-shortname}. TTO includes tasks and example pipelines that allow you to:
+The Scheduling, Scale, and Performance (SSP) Operator integrates {VirtProductName} with {pipelines-shortname}. SSP includes tasks and example pipelines that allow you to:
```

Comment:
Lead-in sentences need to be full sentences. Also, "allow" is a bit too anthropomorphic. SSP functionality doesn't allow anything because it isn't human. Better to rewrite like this: "You can perform the following tasks by using SSP:" It's a complete sentence, active voice, and 2nd person.